examples/step-32/step-32.cc

```diff
@@ -3577,9 +3577,9 @@ namespace Step32
                                 EquationData::R1,
                                 (dim==3) ? 96 : 12,
                                 true);
-    static HyperShellBoundary<dim> boundary;
-    triangulation.set_boundary (0, boundary);
-    triangulation.set_boundary (1, boundary);
+    triangulation.set_all_manifold_ids(0);
+    static SphericalManifold<dim> manifold;
+    triangulation.set_manifold (0, manifold);
```

Comment:
Now the example goes through also in debug mode with no errors. The workaround you wrote is a longer way to write

~~~ C++
 triangulation.set_all_manifold_ids(0);
    triangulation.set_all_manifold_ids_on_boundary(1);
    static SphericalManifold<dim> manifold;
    static HyperShellBoundary<dim> boundary;
    triangulation.set_manifold (0, manifold);
    triangulation.set_manifold (1, boundary);
~~~

which is what I have in right now. The boundary_ids are still left unchanged, so the logi behind the boundary conditions will still work independently.